browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -16936,17 +16936,38 @@ index 0000000000000000000000000000000000000000..45863acc2a61155bdc6db48110a7b196
 +
 +void WebPageInspectorInputAgent::dispatchMouseEvent(const String& type, int x, int y, std::optional<int>&& modifiers, const String& button, std::optional<int>&& buttons, std::optional<int>&& clickCount, std::optional<int>&& deltaX, std::optional<int>&& deltaY, Ref<DispatchMouseEventCallback>&& callback)
 +{
++    if (type != "down" && type != "up" && type != "move") {
```

Comment:
ASSERT_UNUSED in the the beginning were less error-prone actually, let's restore that if it compiles on other platforms